lexmapr/pipeline.py

```diff
@@ -205,134 +256,44 @@ def run(args):
-            ctr += 1
+    spellingDict = get_resource_dict("ScorLex.csv")
+    spellingLowerDict = get_resource_dict("ScorLex.csv", True)
 
     # 15-Get candidate processes from resource in a CSV file format and put in a dictionary to be used further
-    processDict = {}
-    with open(resource_filename('lexmapr.resources', 'candidateProcesses.csv')) as csvfile:
-        ctr = 0
-        readCSV = csv.reader(csvfile, delimiter=',')
-        for row in readCSV:
-            if ctr > 0:  # It gives a first line of headings a skip
-                processTerm = row[0]
-                processExpansion = row[1]
-                processDict[processTerm.strip()] = processExpansion.strip()
-            ctr += 1
+    processDict = get_resource_dict("candidateProcesses.csv")
     
     # 16-Get all semantic tags (e.g.qualities) from resource in a CSV file format and put in a dictionary to be used further
-    qualityDict = {}
-    qualityLowerDict = {}
-    with open(resource_filename('lexmapr.resources', 'SemLex.csv')) as csvfile:              # SEmLex.csv  is a lexicon for semantic tags for qualities, Structure, Furniture, Equipment, Container etc.
-        ctr = 0
-        readCSV = csv.reader(csvfile, delimiter=',')
-        for row in readCSV:
-            if ctr > 0:  # It gives a first line of headings a skip
-                qualityTerm = row[0]
-                qualityExpansion = row[1]
-                qualityDict[qualityTerm.strip()] = qualityExpansion.strip()
-                qualityLowerDict[qualityTerm.strip().lower()] = qualityExpansion.strip()
-            ctr += 1
+    qualityDict = get_resource_dict("SemLex.csv")
+    qualityLowerDict = get_resource_dict("SemLex.csv", True)
 
     # 17-Get all collocations (Wikipedia) from resource in a CSV file format and put in a dictionary to be used further
-    collocationDict = {}
-    with open(resource_filename('lexmapr.resources', 'wikipediaCollocations.csv')) as csvfile:
-        ctr = 0
-        readCSV = csv.reader(csvfile, delimiter=',')
-        for row in readCSV:
-            if ctr > 0:  # It gives a first line of headings a skip
-                collocationTerm = row[0]
-                collocationId = row[1]
-                collocationDict[collocationTerm.strip()] = collocationId.strip()
-            ctr += 1
+    collocationDict = get_resource_dict("wikipediaCollocations.csv")
 
     # 18-Method to get all inflection exception words from resource in CSV file format -Needed to supercede the general inflection treatment
-    inflectionExceptionList = []
-    with open(resource_filename('lexmapr.resources', 'inflection-exceptions.csv')) as csvfile:
-        ctr2 = 0
-        readCSV = csv.reader(csvfile, delimiter=',')
-        for row in readCSV:
-            if ctr2 > 0:
-                abbTerm = row[0]
-                inflectionExceptionList.append(abbTerm.strip().lower())
-            ctr2 += 1
+    inflection_exc_dict = get_resource_dict("inflection-exceptions.csv", True)
```

Comment:
Could we keep the more descriptive variable name `inflection_exception_dict`? Or even just `inflection_exceptions`?